pinot-broker/pom.xml

```diff
@@ -45,6 +45,10 @@
       <groupId>org.apache.pinot</groupId>
       <artifactId>pinot-timeseries-planner</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-core</artifactId>
+    </dependency>
```

Comment:
Why do you need to add this dependency? The marker and MDC classes you use are from slf4j

Reply:
You're right, the used classes belong to `slf4j-api`. I should explicitly add it to the module's `pom.xml` anyway, right? 89b7f5ef7f